content/docs/studio/user-guide/projects-and-experiments/run-experiments.md

```diff
@@ -146,27 +146,59 @@ automatically be created from the new branch to the base branch. Now, click on
 
 ![](https://static.iterative.ai/img/studio/cml_commit.png)
 
-At this point, the new experiment appears in the project's experiment table. If
-you just committed to a new branch, then a new pull request will also have been
-created from the new branch to the base branch.
+## What happens after you submit a new experiment
+
+Iterative Studio will
+[create the Git commit and pull request](#git-commit-and-pull-request-are-created).
```

Comment:
This is still somewhat confusing. We're submitting an experiment but a regular Git commit is created?

Reply:
Yes. The experiment you submit from Studio is not a `dvc exp` experiment. It's a Git commit. We've called it experiment from the beginning, because we expect you to have configured a CI job for model training. We could consider renaming it, but it'll impact many pages in the docs. So, it should become a different issue/PR.